src/python_testing/spec_parsing_support.py

```diff
@@ -328,22 +328,31 @@ def build_xml_clusters() -> tuple[list[XmlCluster], list[ProblemNotice]]:
             clusters[id] = new
 
     # workaround for aliased clusters not appearing in the xml. Remove this once https://github.com/csa-data-model/projects/issues/373 is addressed
-    aliased_clusters = {0x040C: 'Carbon Monoxide Concentration Measurement',
-                        0x040D: 'Carbon Dioxide Concentration Measurement',
-                        0x0413: 'Nitrogen Dioxide Concentration Measurement',
-                        0x0415: 'Ozone Concentration Measurement',
-                        0x042A: 'PM2.5 Concentration Measurement',
-                        0x042B: 'Formaldehyde Concentration Measurement',
-                        0x042C: 'PM1 Concentration Measurement',
-                        0x042D: 'PM10 Concentration Measurement',
-                        0x042E: 'Total Volatile Organic Compounds Concentration Measurement',
-                        0x042F: 'Radon Concentration Measurement'}
-    alias_base_name = 'Concentration Measurement Clusters'
-    for id, alias_name in aliased_clusters.items():
-        base = derived_clusters[alias_base_name]
-        new = deepcopy(base)
-        new.derived = alias_base_name
-        new.name = alias_name
-        clusters[id] = new
+    conc_clusters = {0x040C: 'Carbon Monoxide Concentration Measurement',
+                     0x040D: 'Carbon Dioxide Concentration Measurement',
+                     0x0413: 'Nitrogen Dioxide Concentration Measurement',
+                     0x0415: 'Ozone Concentration Measurement',
+                     0x042A: 'PM2.5 Concentration Measurement',
+                     0x042B: 'Formaldehyde Concentration Measurement',
+                     0x042C: 'PM1 Concentration Measurement',
+                     0x042D: 'PM10 Concentration Measurement',
+                     0x042E: 'Total Volatile Organic Compounds Concentration Measurement',
```

Comment:
@cecille Instead of hard coding all cluster data (ID,name ..etc) inside this script, why don't we read that info from an external file and use it in python script. Now we have a JSON file with cluster data generated from spec @, https://groups.csa-iot.org/wg/matter-csg/document/27290

Reply:
Talked with @cecille, this should he handled with a follow up. Resolving for now.